app/assets/javascripts/admin/addon/mixins/setting-component.js

```diff
@@ -216,8 +217,12 @@ export default Mixin.create({
     }
   }),
 
-  disableSaveButton: computed("validationMessage", function () {
-    return !!this.validationMessage;
+  disableSaveButton: computed("isSaving", "validationMessage", function () {
+    return !!this.validationMessage || this.isSaving;
+  }),
+
+  disableUndoButton: computed("isSaving", function () {
+    return !!this.isSaving;
```

Comment:
We know that `isSaving` is always a boolean, but I'm using double bang here to avoid triggering a linting rule (to use the `reads` macro), which will just make eventual Octane conversion harder.